### ISSUE TITLE
Output lacks final linefeed

### ISSUE THREAD

**Issue by bland328** [mbarkhau/pretty-traceback#3]
In a Linux environment, `pretty-traceback` is fantastic, except that for me it isn't issuing a final linefeed, leaving my prompt appended to the end of the final line of output.

A quick peek at the code suggests to me that this is to be expected, owing to use of `sys.stderr.write()` in `hook.py`, though I'm unclear if this behavior is intentional or not.

Assuming it isn't intentional, and hoping that the linefeed is also desirable and appropriate under Windows, I added `+ '\n'` within `hook.py` thusly to get it working as I'd expect on my system:
```py
def init_excepthook(color):
    def excepthook(exc_type, exc_value, traceback):
        tb_str = formatting.exc_to_traceback_str(exc_value, traceback, color) + '\n'
```

**Comment by mbarkhau** [comment]
If the standard behaviour of python is to have a final linefeed, then I think it's fair to behave the same. I'll be happy to accept a PR if this is the case.

**Comment by mbarkhau** [comment]
Fixed with [2022.1018](https://pypi.org/project/pretty-traceback/2022.1018/)
